lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryMapper.java

```diff
@@ -217,6 +223,9 @@
           + "<if test=\"addAttachmentClassificationNameToSelectClauseForOrdering\">"
           + ", ACNAME "
           + "</if>"
+          + "<if test=\"addWorkbasketNameForOrdering\">"
```

Comment:
should we name this "addWorkbasketNameToSelectClauseForOdering", to be consistent with the if tests for ClassificationName and AttachmentClassificationName? (See TaskQueryImpl comment)